### PR TITLE
Update imported pal headers

### PR DESCRIPTION
**Pull request by JaxLinAMD**

### CODE REVIEW

**Comment by JaxLinAMD** [comment]
retest this please

File: lgc/patch/gfx9/chip/llpcGfx9ConfigBuilder.cpp [hunk 1]
```diff
@@ -115,7 +115,7 @@ void ConfigBuilder::buildPipelineVsFsRegConfig() // [out] Size of register confi
   if (stageMask & shaderStageToMask(ShaderStageVertex)) {
     buildVsRegConfig<PipelineVsFsRegConfig>(ShaderStageVertex, &config);
 
-    SET_REG_FIELD(&config, VGT_SHADER_STAGES_EN, VS_EN, VS_STAGE_REAL);
+    SET_REG_FIELD(&config, VGT_SHADER_STAGES_EN, VS_EN, Pal::Gfx9::Chip::VS_STAGE_REAL);
```
Comment (amdrexu) [inline lgc/patch/gfx9/chip/llpcGfx9ConfigBuilder.cpp:118]: Please add `using namespace Pal::Gfx9::Chip` at the beginning of this file. Thus, we don't have to do so many modifications.

File: lgc/patch/gfx6/chip/llpcGfx6ConfigBuilder.cpp [hunk 2]
```diff
@@ -97,7 +97,7 @@ void ConfigBuilder::buildPipelineVsFsRegConfig() {
   if (stageMask & shaderStageToMask(ShaderStageVertex)) {
     buildVsRegConfig<PipelineVsFsRegConfig>(ShaderStageVertex, &config);
 
-    SET_REG_FIELD(&config, VGT_SHADER_STAGES_EN, VS_EN, VS_STAGE_REAL);
+    SET_REG_FIELD(&config, VGT_SHADER_STAGES_EN, VS_EN, Pal::Gfx6::Chip::VS_STAGE_REAL);
```
Comment (amdrexu) [inline lgc/patch/gfx6/chip/llpcGfx6ConfigBuilder.cpp:100]: Please add `using namespace Pal::Gfx6::Chip` at the beginning of this file.

File: lgc/patch/gfx6/chip/llpcGfx6Chip.cpp [hunk 3]
```diff
@@ -128,11 +128,11 @@ PsRegConfig::PsRegConfig() {
 
 // =====================================================================================================================
 // Gets the starting register ID of SPI_PS_INPUT_CNTL.
-unsigned PsRegConfig::getPsInputCntlStart() { return mmSPI_PS_INPUT_CNTL_0; }
+unsigned PsRegConfig::getPsInputCntlStart() { return Pal::Gfx6::Chip::mmSPI_PS_INPUT_CNTL_0; }
```
Comment (amdrexu) [inline lgc/patch/gfx6/chip/llpcGfx6Chip.cpp:131]: Please add using namespace Pal::Gfx6::Chip at the beginning of this file.

File: lgc/patch/gfx9/chip/llpcGfx9Chip.h [hunk 4]
```diff
@@ -94,18 +94,18 @@ namespace Gfx9 {
...
       _reg##_VAL.u32All = 0;                                                                                           \
     } else {                                                                                                           \
       INIT_REG_TO_INVALID(_reg);                                                                                       \
     }                                                                                                                  \
   }
 
-// GFX10.1 plus
-#define INIT_REG_GFX10_1_PLUS(_gfxMajor, _gfxMinor, _reg)                                                              \
+// Apu09_1xPlus
+#define INIT_REG_APU09_1XPLUS(_gfx, _reg)                                                                              \
```
Comment (amdrexu) [inline lgc/patch/gfx9/chip/llpcGfx9Chip.h:105]: INIT_REG_APU09_1XPLUS -> INIT_REG_APU09_1X_PLUS

File: lgc/patch/gfx9/chip/llpcGfx9Chip.h [hunk 5]
```diff
@@ -94,18 +94,18 @@ namespace Gfx9 {
...
+      _reg##_ID = Pal::Gfx9::Chip::Gfx10::mm##_reg;                                                                    \
       _reg##_VAL.u32All = 0;                                                                                           \
     } else {                                                                                                           \
       INIT_REG_TO_INVALID(_reg);                                                                                       \
     }                                                                                                                  \
   }
 
-// GFX10.1 plus
-#define INIT_REG_GFX10_1_PLUS(_gfxMajor, _gfxMinor, _reg)                                                              \
+// Apu09_1xPlus
```
Comment (amdrexu) [inline lgc/patch/gfx9/chip/llpcGfx9Chip.h:104]: Apu09_1xPlus -> APU09.1x plus

File: lgc/patch/gfx9/chip/llpcGfx9Chip.h [hunk 6]
```diff
@@ -142,11 +142,11 @@ namespace Gfx9 {
...
 // Adds an entry for the map from register ID to its name string
-#define ADD_REG_MAP(_reg) RegNameMap[mm##_reg * 4] = #_reg;
+#define ADD_REG_MAP(_reg) RegNameMap[Pal::Gfx9::Chip::mm##_reg * 4] = #_reg;
 
-#define ADD_REG_MAP_GFX9(_reg) RegNameMapGfx9[Gfx09::mm##_reg * 4] = #_reg;
-#define ADD_REG_MAP_GFX10(_reg) RegNameMapGfx10[Gfx10::mm##_reg * 4] = #_reg;
-#define ADD_REG_MAP_GFX10_1_PLUS(_reg) RegNameMapGfx10[Gfx101Plus::mm##_reg * 4] = #_reg;
+#define ADD_REG_MAP_GFX9(_reg) RegNameMapGfx9[Pal::Gfx9::Chip::Gfx09::mm##_reg * 4] = #_reg;
+#define ADD_REG_MAP_GFX10(_reg) RegNameMapGfx10[Pal::Gfx9::Chip::Gfx10::mm##_reg * 4] = #_reg;
+#define ADD_REG_MAP_APU09_1XPLUS(_reg) RegNameMapGfx10[Pal::Gfx9::Chip::Apu09_1xPlus::mm##_reg * 4] = #_reg;
```
Comment (amdrexu) [inline lgc/patch/gfx9/chip/llpcGfx9Chip.h:149]: ADD_REG_MAP_APU09_1XPLUS -> ADD_REG_MAP_APU09_1X_PLUS

**Pull request closed without merging**